src/Illuminate/Pagination/AbstractPaginator.php

```diff
@@ -305,14 +305,15 @@ public static function currentPathResolver(Closure $resolver)
 	/**
 	 * Resolve the current page or return the default value.
 	 *
+	 * @param  string  $pageName
 	 * @param  int  $default
 	 * @return int
 	 */
-	public static function resolveCurrentPage($default = 1)
+	public static function resolveCurrentPage($pageName = 'page', $default = 1)
```

Comment:
I noticed this, I searched all calls to `resolveCurrentPage()` on the code and none is passing that parameter yet.